contrib/signet/README.md

```diff
@@ -23,17 +23,16 @@ miner
 
 You will first need to pick a difficulty target. Since signet chains are primarily protected by a signature rather than proof of work, there is no need to spend as much energy as possible mining, however you may wish to choose to spend more time than the absolute minimum. The calibrate subcommand can be used to pick a target appropriate for your hardware, eg:
 
-    cd src/
-    MINER="../contrib/signet/miner"
-    GRIND="./bitcoin-util grind"
+    MINER="./contrib/signet/miner"
+    GRIND="./build/src/bitcoin-util grind"
```

Comment:
nit:
```suggestion
    GRIND="./build/src/bitcoin-util grind" #assuming the build directory is named `build`
```

Reply:
This is the build directory used by the [unix](https://github.com/bitcoin/bitcoin/blob/master/doc/build-unix.md#to-build) and [macOS](https://github.com/bitcoin/bitcoin/blob/master/doc/build-osx.md#2-compile) build docs. IMO it's therefore not necessary to mention that here. If people are clever enough to change the build directories they'll also understand that this changes the location of binaries.

Reply:
ok, just to be consistent on other places in the documentation where this is clarified